crates/polars-core/src/datatypes/dtype.rs

```diff
@@ -377,3 +377,9 @@ pub fn merge_dtypes(left: &DataType, right: &DataType) -> PolarsResult<DataType>
         _ => polars_bail!(ComputeError: "unable to merge datatypes"),
     })
 }
+
+#[cfg(feature = "dtype-categorical")]
+pub fn create_enum_data_type(categories: Utf8Array<i64>) -> DataType {
+    let rev_map = RevMapping::build_enum(categories.clone());
+    DataType::Categorical(Some(Arc::new(rev_map)))
+}
```

Comment:
Since this util is only used in `py-polars`, maybe it should be defined there? Or is this something we want in our public Rust API?